### PR TITLE
[feature] Added support for Ubuntu 22 #376

### LINKED ISSUE #376: [feature] Add support for Ubuntu 22
Add support for the new Ubuntu 22.

### PR DESCRIPTION
**Pull request by Aryamanz29**
~~Depends on : https://github.com/openwisp/ansible-ow-influxdb/pull/10~~
Closes #376

### CODE REVIEW

File: tasks/freeradius.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   become: true
   when:
     - ansible_distribution|string == 'Ubuntu'
-    - ansible_distribution_release|string == 'focal'
+    - ansible_distribution_release|string == 'focal' or ansible_distribution_release|string == 'jammy'
```
Comment (Aryamanz29) [inline tasks/freeradius.yml:9]: Freeradius package, currently not available for `Ubuntu 22.04 (Jammy Jellyfish)`. 
See this :point_right:  https://networkradius.com/packages/

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
@Aryamanz29 can you please check what is going on here:

```
  TASK [ansible-openwisp2 : SQL configuration] ***********************************
  fatal: [openwisp2-ubuntu2204]: FAILED! => changed=false 
    checksum: e9e63f34902d5f2a10bd0841cbd95bb821f7e692
    msg: Destination directory /etc/freeradius/mods-available does not exist
```

**Comment by Aryamanz29** [comment]
> @Aryamanz29 can you please check what is going on here:
> 
> ```
>   TASK [ansible-openwisp2 : SQL configuration] ***********************************
>   fatal: [openwisp2-ubuntu2204]: FAILED! => changed=false 
>     checksum: e9e63f34902d5f2a10bd0841cbd95bb821f7e692
>     msg: Destination directory /etc/freeradius/mods-available does not exist
> ```
Just checked in `ubuntu22` image
```bash
# ls /etc/freeradius/
3.0
```
because of that, this task fails with error msg `/etc/freeradius/mods-available does not exist`

File: defaults/main.yml [hunk 2]
```diff
@@ -159,7 +159,7 @@ openwisp2_radius_delete_old_postauth: 365
 openwisp2_radius_delete_old_radacct: 365
 openwisp2_radius_allowed_hosts: ["127.0.0.1"]
 openwisp2_freeradius_install: true
-freeradius_dir: /etc/freeradius
+freeradius_dir: "{{ '/etc/freeradius/3.0' if ansible_distribution_release|string == 'jammy' else '/etc/freeradius/' }}"
```
Comment (Aryamanz29) [inline defaults/main.yml:162]: For **ubuntu22 jammy**, `freeradius_dir` should be `/etc/freeradius/3.0`

**Review by nemesisdesign — CHANGES_REQUESTED** [review]
Looks good! Can you update the line of the README where it says:

```
Tested on Debian (Buster, Bullseye), Ubuntu (18/20 LTS).
```

We can write "Ubuntu (18/20/22 LTS)".

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**